README.md

```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for homework 1
+
+Repository for first CFSS homework assignment
+
+hello! my name is Jenny Lu and I'm a PhD student in the **Psychology** Department.
+
+*A few facts about myself
+
+1. I'm originally from Boston
+2. I love to cook using different cooking techniques like sous vide
+3. I like to do processes that involve fermentation such as making yogurt, bread, and beer
+
+I also run a workshop called ["Modalities of Lanugage"] (https://voices.uchicago.edu/modalitiesoflanguage/) where students and faculty come together to discuss interesting current research on sign language and gesture.
+*Come by and check it out!*
+
+![](Hw1 final try/puppy.jpg)
```

Comment:
Images can be tricky; the keys are making sure your .jpg is saved within the repo and pushed to Github as well. Also try getting in the hang of having no spacing in your file titles, so that the file path can be clearly read by markdown.